cinema/models.py

```diff
@@ -41,6 +45,14 @@ class Movie(models.Model):
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, blank=True, upload_to="uploads/")
+
+    def create_custom_path(self, instance, filename):
+        _, extension = os.path.splitext(filename)
+        return os.path.join(
+            "uploads/images/",
+            f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
+        )
```

Comment:
```suggestion
        return os.path.join(
            "uploads", "images",
            f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
        )
```

Read how this works and why